webapp/src/Command/ImportEventFeedCommand.php

```diff
@@ -177,11 +177,11 @@ protected function configure(): void
      */
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        // Disable SQL logging if we do not run explicitly in debug mode.
-        // This would cause a serious memory leak otherwise since this is a
-        // long running process.
-        if (!$this->debug) {
-            $this->em->getConnection()->getConfiguration()->setSQLLogger(null);
+        // Disable SQL logging and profiling. This would cause a serious memory leak otherwise
+        // since this is a long running process.
+        $this->em->getConnection()->getConfiguration()->setSQLLogger();
+        if ($this->profiler) {
+            $this->profiler->disable();
```

Comment:
This is only for the php process running the import command?

Reply:
It is for the current PHP process yes, just like we do in the judgehost API.